Makefile

```diff
@@ -0,0 +1,15 @@
+PORT := ${or ${PORT},${PORT},8000}
+SSH_AUTH_SOCK := /run/host-services/ssh-auth.sock
```

Comment:
Note: this is the version required for it to work seamlessly on OS X (the actual  `SSH_AUTH_SOCK` env var value won't work). But we should adapt for other systems.